gotools.mk

```diff
@@ -0,0 +1,14 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+GOTOOLS = $(shell grep '_' tools/tools.go | cut -d\" -f2 | rev | cut -d/ -f1 | rev)
```

Comment:
This is clever but pretty fragile. It's looking for any line with an underscore in it. This is assuming we're only going to get package imports but you never know ...

Why not just list the targets explicitly? The idea of tools.go is to enable go.mod to version the deps; it's not really meant to be a tools manifest.